src/main/webapp/js/omniComment.js

```diff
@@ -86,16 +86,16 @@ $(document).ready(function() {
             $('#no-comment-panel').show();
             // if all is checked, show giver and status for better user experience
             if (!$('#panel_all').prop('checked')) {
-                $('#giver_all').parent().parent().hide();
-                $('#status_all').parent().parent().hide();
+                $('#giver_all').closest('.col-sm-4').hide();
```

Comment:
This doesn't have an id either. Only, `<div class="col-sm-4">`. I had originally wanted to add a class to the div to get `<div class="col-sm-4 filter-options">` but I removed it to avoid godmode changes. Would doing adding the `filter-options` class be preferred?